### PR TITLE
Add note about returning current results

### PR DESCRIPTION
**Pull request by AlexAndorra**
Making sure the API is clearer in the README for users who want to get the results from the background sampler

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,7 +105,10 @@ sampler.pause()
 sampler.resume()
 
 # Wait for the sampler to finish (up to timeout seconds)
-# sampler.wait(timeout=0.1)
+sampler.wait(timeout=0.1)
+# Note that not passing any timeout to `wait` will
+# directly save and return the results:
```
Comment (twiecki) [inline README.md:110]: the current results?

Comment (AlexAndorra) [reply README.md]: Yep. Just updated so that's even clearer

File: README.md [hunk 2]
```diff
@@ -105,7 +105,10 @@ sampler.pause()
 sampler.resume()
 
 # Wait for the sampler to finish (up to timeout seconds)
-# sampler.wait(timeout=0.1)
+sampler.wait(timeout=0.1)
+# Note that not passing any timeout to `wait` will
+# directly save and return the current results once the sampler finishes:
```
Comment (twiecki) [inline README.md:110]: Apologies, that was actually a legit question, I think I was confused by "directly".
```suggestion
# block until the sampler finishes and return the InferenceData object:
```

**Comment by AlexAndorra** [comment]
Good point -- updated 👌

**Review by twiecki — APPROVED** [review]

**Pull request merged**